packages/core-input/readme.md

```diff
@@ -77,15 +77,22 @@ Always use `coreInput.escapeHTML(String)` to safely render data from API or user
 ```js
 import coreInput from '@nrk/core-input'
 
-coreInput(String|Element|Elements, {    // Accepts a selector string, NodeList, Element or array of Elements
-  open: Boolean,                        // Optional. Defaults to value of aria-expanded. Use to force open state
-  content: String,                      // Optional. Set String of HTML content. HTML is used for full flexibility on markup
-  ajax: String                          // Optional. Fetch external data, example: "https://search.com?q={{value}}"
+coreInput(                              // Initializes input element
+  String|Element|Elements,              // Accepts a selector string, NodeList, Element or array of Elements
+  String|Object {                       // Optional. String sets content HTML, object sets options
+    open: Boolean,                      // Use to force open state. Defaults to value of aria-expanded.
+    content: String,                    // Sets content HTML. HTML is used for full flexibility on markup
+    limit: Number                       // Limit the maximum number of results in list.
```

Comment:
```suggestion
    limit: Number,                      // Limit the maximum number of results in list.
```

Reply:
Burde vi også tydeliggjøre at `limit` ikke påvirker hvor mange resultater som blir rendret fra AJAX, siden man selv er ansvarlig for å lage markup her?